WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderMenuAdapter.java

```diff
@@ -50,8 +59,42 @@ public long getItemId(int position) {
         return mMenuItems.get(position).getType().ordinal();
     }
 
+    @Override
+    public int getViewTypeCount() {
+        return 2;
+    }
+
+    @Override
+    public int getItemViewType(int position) {
+        return (mMenuItems.get(position).getType() == ReaderPostCardActionType.SPACER_NO_ACTION) ? 0 : 1;
```

Comment:
Warning (⚠️): Consider using the constants defined above instead of hardcoded values (`TYPE_SPACER` instead of `0` and `TYPE_CONTENT` instead of `1`). Also, this way you get to use the currently unused `TYPE_CONTENT` constant.